actions/utils.ts

```diff
@@ -4,56 +4,125 @@ import assert = require("assert");
 import { ethers } from "ethers";
 import { ConnectionInfo, Logger } from "ethers/lib/utils";
 import { OrderStatus, Registry } from "./register";
+
 import Slack = require("node-slack");
+import {
+  init as sentryInit,
+  startTransaction as sentryStartTransaction,
+  Transaction as SentryTransaction,
+} from "@sentry/node";
+import { CaptureConsole as CaptureConsoleIntegration } from "@sentry/integrations";
+
+// const Sentry = require("@sentry/node");
```

Comment:
```suggestion
```

Reply:
I will delete in a follow up PR (small refactor)